pom.xml

```diff
@@ -610,7 +610,7 @@
     <external.protoc.version>${external.protobuf.version}</external.protoc.version>
     <!--Version of protobuf that hbase uses internally (we shade our pb)
          Must match what is out in hbase-thirdparty include. -->
-    <internal.protobuf.version>4.26.1</internal.protobuf.version>
```

Comment:
Looks like we missed syncing the internal protobuf version when we bumped to hbase-thirdparty-4.1.8.